biz_days.py

```diff
@@ -117,10 +120,12 @@ def weekdays(dow1, dow2):
     if dow1 < 0 or dow1 > 6 or dow2 < 0 or dow2 > 6:
         raise ValueError
     if dow1 == dow2:
-        return 1 if dow1 < 5 else 0
+        # return 1 if dow1 < 5 else 0
+        # Why the above? There's not 1 weekday BETWEEN Mon & Mon
+        return 0
     d2 = dow2 if dow2 > dow1 else dow2 + 7
-
-    return len(tuple(i for i in range(dow1, d2 + 1) if i not in (5, 6)))
+    # Count the desired number of days without creating intermediate tuples
```

Comment:
If it turns out that this approach is faster, then we can keep the change but nix the comment.